src/components/Popup.js

```diff
@@ -31,6 +32,14 @@ export default class Popup {
     }
   }
 
+  renderLoading(isLoading) {
+    if (isLoading) {
+      this._saveButton.textContent = "Saving...";
+    } else {
+      this._saveButton.textContent = "Save";
+    }
```

Comment:
* `this._saveButton` is not needed in `Popup` because the image modal doesn't have it. Universal classes can't have elements that don't exist in all modals

`renderLoading` should be in `PopupWithForm` due to the same reason